src/desktop/apps/auction/components/Layout.js

```diff
@@ -27,8 +28,16 @@ function Layout(props) {
 
   const b = block("auction-Layout")
 
+  const Modal = ConfirmRegistrationModal
+  // const Modal =
+  //   typeof window !== "undefined" &&
+  //   window.location.pathname.match("/confirm-registration")
+  //     ? ConfirmRegistrationModal
+  //     : null
+
   return (
     <div className={b()}>
+      {Modal && <Modal />}
```

Comment:
I understand that this is WIP, but this always evaluates to `true` and you do not need the `&&` guard.